spec/features/transactions_filter_spec.rb

```diff
@@ -254,13 +254,13 @@
 
     context 'when custom' do
       let!(:transaction)  { Timecop.travel(2013,12,12) {
-        create :transaction, bank_account: ba } }
+        create :transaction, bank_account: ba, date: Time.now } }
```

Comment:
Timecop.travel(1.month.ago) { create :transaction...} Not working with date: Time.now in factory